lineman/app/components/thread_preview_collection/thread_preview_collection.scss

```diff
@@ -1,4 +1,4 @@
-.thread-previews-container {
+.inbox-page__groups .thread-previews-container, .dashboard-page__collections .thread-previews-container {
```

Comment:
It's because the thread-previews-container on Inbox and Dashboard behave
like cards, but on the Group page it lives inside a card. Not sure the best
way to solve that

On Thu, Oct 8, 2015 at 8:30 AM Robert Guthrie <notifications@github.com>
wrote:

> In
> lineman/app/components/thread_preview_collection/thread_preview_collection.scss
> <https://github.com/loomio/loomio/pull/2548#discussion_r41434862>:
>
> > @@ -1,4 +1,4 @@
> > -.thread-previews-container {
> > +.inbox-page__groups .thread-previews-container, .dashboard-page__collections .thread-previews-container {
>
> This is almost great. I'm wondering why there are qualifiers here. I mean,
> are there thread-previews-containers which won't work with this css? if so,
> why? If not.. then we should be able to drop the qualifiers here and just
> use .thread-previews-container as the selector
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2548/files#r41434862>.
>